include/unifex/receiver_concepts.hpp

```diff
@@ -195,6 +199,18 @@ inline constexpr bool is_receiver_cpo_v = is_one_of_v<
     _rec_cpo::_set_error_fn,
     _rec_cpo::_set_done_fn>;
 
+
+// HACK: Approximation for CPOs that should be forwarded through receivers
+// as query operations.
+template <typename T>
+inline constexpr bool is_receiver_query_cpo_v = !is_one_of_v<
+    remove_cvref_t<T>,
+    _rec_cpo::_set_value_fn,
+    _rec_cpo::_set_next_fn,
+    _rec_cpo::_set_error_fn,
+    _rec_cpo::_set_done_fn,
+    _connect::_fn>;
```

Comment:
This isn't exactly what I had in mind. ;-)

Reply:
I know. It was a stop-gap to see if that fixed the issue I was seeing.
I'll replace it with something more opt-in and category-like later.